Assets/Scripts/PlayerCharacter/PlayerController.cs

```diff
@@ -40,11 +48,15 @@ private void Start()
             groundCheckDistance = radius;
         }
```

Comment:
i hate github. i can't comment on "hidden" lines in this file. wtf...

```cs
        private Rigidbody rb;
        private CapsuleCollider coll;
        private DefaultInputActions actions;
        private void Start()
        {
            rb = GetComponent<Rigidbody>();
            rb.useGravity = false;
            rb.freezeRotation = true;
            coll = GetComponent<CapsuleCollider>();
            var radius = coll.radius;
            groundCheckRadius = radius * 0.99f; // 0.99 - padding to make spherecast detect floor
            groundCheckDistance = radius;
        }
```
`private CapsuleCollider coll;` is used only inside `Start` method, you can remove this field.